src/resources/views/base/inc/scripts.blade.php

```diff
@@ -63,3 +63,5 @@
         location.hash = e.target.hash.replace("#tab_", "#");
     });
 </script>
+
+@include('crud::inc.error_frame')
```

Comment:
Let's call it `ajax_error_frame` to be even more specific.

Reply:
Did it 👌